.github/workflows/ci.yml

```diff
@@ -39,6 +46,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
           miniforge-variant: Mambaforge
           use-mamba: true
+          channels: conda-forge/label/python_rc,conda-forge
```

Comment:
Nice, TIL @henryiii!

Reply:
I figured it out today too. :) I've been seeing my feedstocks being rebuilt for 3.13, so I knew it had to be somewhere.